src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/IModelBindingContext.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Mvc.ModelBinding.Validation;
+
+namespace Microsoft.AspNetCore.Mvc.ModelBinding
+{
+    /// <summary>
+    /// A context that contains operating information for model binding and validation.
```

Comment:
(Realize you copied many of these comments from `ModelBindingContext` but ...) Bit confusing because while this context includes some information gathered for later validation, it is not used during that process.